packages/app/src/index.ts

```diff
@@ -17,10 +17,10 @@ export type { DataContextValue } from './providers/DataProvider';
 
 // Undocumented
 export { assertEnvVar } from '@h5web/shared/guards';
+export { assertNonNull } from '@h5web/shared/guards';
 
 // Undocumented (for @h5web/h5wasm)
 export { default as DataProvider } from './providers/DataProvider';
 export { DataProviderApi } from './providers/api';
 export type { ValuesStoreParams } from './providers/models';
-export { getNameFromPath, getValueOrError } from './providers/utils';
```

Comment:
I move `getNameFromPath` to the shared package so I can later import it in a web worker. For some reason, importing it from here led to pending promises as explained in https://github.com/silx-kit/h5web/issues/1582#issuecomment-2034729074